site/gatsby-site/src/components/forms/IncidentReportForm.js

```diff
@@ -110,8 +110,74 @@ const IncidentReportForm = ({ incident, onUpdate, onSubmit }) => {
 
   const isEditMode = incident && !!incident.incident_id;
 
+  const addToast = useToastContext();
+
+  const [parsingNews, setParsingNews] = useState(false);
+
+  const coldStartToast = () => {
+    addToast({
+      message: <>Sometimes fetching news info may take a while...</>,
+      severity: SEVERITY.warning,
+    });
+  };
+
+  const parseNewsUrl = async (newsUrl) => {
+    setParsingNews(true);
+    const timeout = setTimeout(coldStartToast, 20000);
+
+    try {
+      const url = `https://z14490usg0.execute-api.us-east-1.amazonaws.com/default/parseNews?url=${encodeURIComponent(
```

Comment:
@cesarvarela I am going to merge this PR, but let's move this connection string into the config file. I want to keep all the lambda functions and integrations in one place.